app/controllers/user.rb

```diff
@@ -0,0 +1,115 @@
+require 'erb'
+require 'sinatra/flash'
+require 'pony'
+require 'json'
+
+module  Sinatra
+  module App
+    module SignUp
```

Comment:
This controller is handling much more than **Sign up**. I suggest that you split it into smaller controllers that handle specific aspects. For example, this can be broken down into:
- Sessions (log in/ log out)
- Admin Dashboard etc.